migrations/4_genesis.js

```diff
@@ -23,7 +23,8 @@ module.exports = function(deployer, network, accounts) {
 
         let genesisManager
 
-        if (!lpDeployer.isLiveNetwork(network)) {
+        if (!lpDeployer.isProduction(network)) {
```

Comment:
If we are not deploying to main net and we are not doing a dry run on a test network we send the company supply to the deployment account instead of the bank multisig so the deployment account can transfer tokens wherever necessary - this is primarily to make it easier to automate the process of funding the faucet without having to edit the config file